Add osu!taiko's `swell` (#14583)
ppy/osu-wiki: wiki/Gameplay/Hit_object/Swell/en.md

---
tags:
  - hit objects
---

# Swell

> ![](/wiki/shared/Taiko_spinner.jpg "osu!taiko swell")

Reviewer: missing alt text

```suggestion
![An osu!taiko swell](/wiki/shared/Taiko_spinner.jpg "osu!taiko swell")
```

> The **swell** is a circular [hit object](/wiki/gameplay/Hit_object) in [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap) looking like a whirl. Upon reaching the white judgement circle on the left side of the conveyer belt, the swell will be displayed as a big circle, taking up the entire screen. At this point, players need to alternate between hitting the drum's center and rim as fast as possible until the swell disappears.

Reviewer: ```suggestion
The **swell** is a circular [hit object](/wiki/gameplay/Hit_object) in [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap) looking like a whirl. Upon reaching the white judgement circle on the left side of the conveyer belt, the swell will be displayed as a big circle, taking up the entire screen. At this point, players need to alternate between hitting the drum's centre and rim as fast as possible until the swell disappears.
```

It does not matter which part of the drum is hit first. Hitting the same drum part on repeat will not progress the swell. Failing to complete the swell will incur a health penalty, but will not break combo.

## Judgement

> Swells do not increment the combo counter, and do not recover the health bar in any way. Each successful hit on a swell only gives a static 300 score and increments the `BONUS` judgement, while a successful completion of a swell will reward the same amount of score as a finisher hit.

Reviewer: just bonus score is the correct parlance

```suggestion
Swells do not increment the combo counter, and do not recover the health bar in any way. Each successful hit on a swell only gives a static 300 bonus score, while a successful completion of a swell will reward the same amount of score as a finisher hit.
```

## Trivia

- In [Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin), swells are called denden.

> - Due to their similar look to osu!'s [spinners](/wiki/Gameplay/Hit_object/Spinner), swells are sometimes called spinners too.

Reviewer: pretty sure this is more due to the underlying object type being a spinner and because osu!taiko uses the osu! editor. it's the same story in other game modes

Author: Does this fit more? (might be too broad)

```suggestion
- Due to technical reasons, swells are sometimes called spinners too.
```

Reviewer: was thinking more like this

```suggestion
- Due to technically being the same object type as [spinners](/wiki/Gameplay/Hit_object/Spinner) in `.osu` files and other game modes sharing the osu! mode's editor in osu!(stable), swells are sometimes called spinners too.
```
